### DISCUSSION TITLE
Issues and Features

### DISCUSSION THREAD

**Discussion by ojvaj1** [K3vinb5/Unyo#20] [Ideas]
The app looks really really promising, I love the auto import from Anilist and the UI looks clean.

Some changes that would make the app even better.

### Issues that have to be fixed.

1. Problem with going to the next chapter when reading manga. Right now you have to go in and out to go to the next chapter.

### Features for settings.
1. Select a default source for the manga/anime you are reading/watching.
2. Default/global manga reading settings for all mangas you read. 
3. More manga sources [MangaSee](https://mangasee123.com/)/[MangaLife ](https://manga4life.com/)NepNep Sources, [ComicK](https://comick.io/), [MangaDex](https://mangadex.org/), [Tcb Scans](https://tcbscans.me/) some of my favourites (I know you are working on it)
4. Update all sources so it has the latest chapters/episodes from a specific source
5. Sort chapters/episodes in different ways (ex. normal, reverse, by date added)

### Features for manga reader.
1. Continue reading button
2. Optional spacing between pages
3. That you can mark pages as "double/wide panel/grouped panel" and "single/separate panel" and always separate the first page/double page offset. (Marking a page as a single panel fixes the alignment for the panel spread so it displays it properly like the author intended it to be viewed. Not a huge deal but with some manga panels it really makes a difference what panels you see at the same time.)*
6. Crop borders (not my favourite but people like it)
7. Grayscale
8. Background colour (ex. auto, black, white, grey)
9. Fullscreen reader. (hideUI)
10. Pagenumber (if you read with hidden UI)
11. Page dim

### Features for anime player.
1. Continue watching button
2. At what point to mark the episode as seen (ex. 70% 85% 100%)
3. Intro/auto skip
4. Default skip intro length (in sec)
5. Default skip forward length (in sec)
6. Default playback speed)
7. Aniskip/Auto skip.
8. Default prefered dub/sub language
9. Filler skip

### Not important features.

1. Manga page saving/sharing
2. Downloading manga chapters/anime episodes
3. DiscordRP

There are some issues and features that I came to think of when first using the app. Great looking app I think it has massive potential keep up the great work!

**Comment by K3vinb5** [comment]
Hi, thanks for the feedback! Thank you so much for all the suggested features and issues as soon as I finish this next version I will give more attention to the manga and finally add some more sources, I have been implementing a modular extensions system (just for the anime, later manga, I'm sorry manga people) and it's almost done now, expect a lot of your suggestions to be incorporated in next versions, again thanks for the feedback and suggestions, if you have more please answer this comment, it's always good for me to receive feedback and view the perspective from other's point of view :D .

**Comment by ojvaj1** [comment]
One more real sweat feature would be in-app updates, but it's not a prior to do just something to have for a future goal or if you got time over :D I will do my worst with the feedback hehe :)

**Comment by ojvaj1** [comment]
Some more quality-of-life features,
Update mangas/anime on startup (look for new chapters/episodes from a selected source)
Show how many unread/seen chapters/episodes you left.

**Comment by K3vinb5** [comment]
Hii, thanks for you feedback you've been nothing but awesome, I was on vacations for the last 2 weeks but now that I'm back I'll start right away :D. I have some questions about this last suggestion would work but when the time comes you may explain yourself a bit better hehehe.

**Comment by ojvaj1** [comment]
For the "Show how many unread/seen chapters/episodes you left."
![image](https://github.com/user-attachments/assets/a8b396de-1d5f-467c-bf0a-a701bc34c593)
It's like this just a number to show how many chapters/episodes you have unread/unseen.

And for "Update mangas/anime on startup (look for new chapters/episodes from a selected source)" many apps have that you need to update the sources to get the new chapters to show. 

But for Houdoku (another manga reader) they have a "check for updates on startup" and that's the feature I would love to have. 
![image](https://github.com/user-attachments/assets/d994c7c4-0937-43d7-9ecd-bea62275d8ba)

Hope that cleared things up :)

**Comment by K3vinb5** [comment]
Hi, I added a bunch of your requests on this last version there is still some I agree with and would like to add, thanks for all the suggestions, you've been great!
